examples/renv-shiny/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM rhub/r-minimal:4.2.3-patched
```

Comment:
You'll need a newer version, one that is in the README, because 4.2.3-patched was built a year ago and does not have `installr -e`. E.g. `4.3.2` should be good.

Reply:
Using the latest version now